org.eclipse.scout.rt.platform/pom.xml

```diff
@@ -40,7 +40,12 @@
     <dependency>
       <groupId>org.quartz-scheduler</groupId>
       <artifactId>quartz</artifactId>
-   </dependency>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.scout.rt</groupId>
```

Comment:
Add comment why this dependency is added (i.e. "Dependency is onl< required to enable JSON parsing within PropertiesHelper without using IDataObjectMapper, org.eclipse.scout.json must not be used for other scenarios). Something like this.